### PR TITLE
Mobile accessability fix

### PR DESCRIPTION
**Pull request by GoodBoyDigital**
Added a touch 'hook' div to activate the accessibility layer if a screen reader is being used on mobile.

### CODE REVIEW

File: src/accessibility/AccessibilityManager.js [hunk 1]
```diff
@@ -124,6 +152,7 @@ AccessibilityManager.prototype.activate = function()
  */
 AccessibilityManager.prototype.deactivate = function()
 {
+	return;
```
Comment (englercj) [inline src/accessibility/AccessibilityManager.js]: ?

File: src/accessibility/AccessibilityManager.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 var core = require('../core');
+var  Device = require('ismobilejs');
```
Comment (englercj) [inline src/accessibility/AccessibilityManager.js]: New dependency? Probably should add it to `package.json`.

File: src/accessibility/AccessibilityManager.js (hunk 1 above)
Comment (GoodBoyDigital) [inline src/accessibility/AccessibilityManager.js]: good catch! fixed now..

**Comment by GoodBoyDigital** [comment]
@andyish is there a way to check if a mobile device has a screen reader enabled?

File: src/accessibility/AccessibilityManager.js (hunk 2 above)
Comment (GoodBoyDigital) [inline src/accessibility/AccessibilityManager.js]: yup its added to the package.json

**Comment by andyish** [comment]
@GoodBoyDigital No, not that we're aware of. Could leave it to the developer thats using it check. 

With accessibility in games on mobile devices we have a setting to enable it

File: src/accessibility/AccessibilityManager.js (hunk 2 above)
Comment (englercj) [inline src/accessibility/AccessibilityManager.js]: Now I see it in dev branch package.json, ignore me :P

**Pull request merged**